drivers/scheduler/k8s/k8s.go

```diff
@@ -1999,11 +1992,11 @@ func (k *K8s) resizePVCBy1GB(ctx *scheduler.Context, pvc *v1.PersistentVolumeCla
 	}
 	sizeInt64, _ := storageSize.AsInt64()
 	vol := &volume.Volume{
-		ID:        string(pvc.UID),
-		Name:      pvc.Name,
-		Namespace: pvc.Namespace,
-		Size:      uint64(sizeInt64),
-		Shared:    k.isPVCShared(pvc),
+		ID:          string(pvc.UID),
+		Name:        pvc.Name,
+		Namespace:   pvc.Namespace,
+		ResizedSize: uint64(sizeInt64),
```

Comment:
why add a new field called ResizedSize? I don't see you using it anywhere.

This adds confusion since there is already a field called Size. Which does does the user of the API use? Is Size implicitly outdated initial size and should I stop using it?